lib/main/atom/atomConfig.ts

```diff
@@ -17,13 +17,19 @@ class Config {
             type: 'boolean',
             default: false
         },
+        modulePathToProjectRoot: {
+            title: 'Show module path suggestion relative to project root.',
+            type: 'boolean',
+            default: true
+        },
```

Comment:
that's true, I agree it's better to have the suggestion relative to typescript project root.